auth0/rules/auth0-provider-with-history.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
The auth0 top-level folder is for code that goes into the auth0 website, is this needed in the webapp? If so I would move it there